client/src/plugins/camunda-plugin/start-instance-tool/StartInstanceTool.js

```diff
@@ -414,19 +414,24 @@ export default class StartInstanceTool extends PureComponent {
    handleStartError(tab, error) {
      const {
        log,
-       displayNotification
+       displayNotification,
+       triggerAction
      } = this.props;
 
      const logMessage = {
        category: 'start-instance-error',
-       message: error.problems || error.message
+       message: error.problems || error.message,
+       silent: true
      };
 
+     log(logMessage);
+
      const content = <button
-       onClick={ ()=> log(logMessage) }>
+       onClick={ () => triggerAction('open-log') }>
        See the log for further details.
      </button>;
 
+
```

Comment:
Sorry about being picky but let's remove this unnecessary double spacing here

Reply:
One day I'll prove my IDE is sentient and doesn't have my interest at heart 😆, will remove.